client/src/app/pages/applications/analysis-wizard/__tests__/analysis-wizard.test.tsx

```diff
@@ -140,7 +140,8 @@ describe("<AnalysisWizard />", () => {
     await waitFor(() => expect(nextButton).toHaveAttribute("disabled", ""));
   });
 
-  it("can run analysis on applications with a binary definition using defaults", async () => {
+  //TODO
+  it.skip("can run analysis on applications with a binary definition using defaults", async () => {
```

Comment:
What's the issue here ?

Reply:
This test relies on mock data that we are now pulling from the API. IMO we should leave the end to end stuff to the QE tests and use RTL for testing that individual hooks/components are working as expected.